src/app/logic/processGame.js

```diff
@@ -0,0 +1,146 @@
+import Answers from '../components/Answers';
+import VisualImage from '../components/VisualImage';
+import { getQuestion, initGame, isGameInitialized } from './quiz';
+
+class Game {
+  getTimeLeft() {
+    return this.timeLeft;
+  }
+
+  getMode() {
+    return this.mode;
+  }
+
+  getRunning() {
+    return this.running;
+  }
+
+  getQuestions() {
+    return this.questions;
+  }
+
+  reduceTime() {
+    this.timeLeft -= 0.5;
+  }
+
+  addQuestion(question) {
+    this.questions.push(question);
+  }
+
+  newGame(mode) {
+    this.timeLeft = 5 || process.env.QUIZ_MAX_TIME_SECONDS;
+    this.questions = [];
+    this.mode = mode;
+    this.running = false;
+  }
+
+  changeRunningFlag() {
+    this.running = !this.running;
+  }
+
+  generateObjectForModal() {
+    return {
+      mode: this.mode,
+      questions: this.questions,
+    };
+  }
+}
+
+const game = new Game();
+
+const spreadQuestion = (question) => {
+  // TODO: send question to computer player
+  VisualImage(question.image);
+  Answers(
+    question.answers,
+    question.rightAnswer,
+    // Eslint warn. Use before definition
+    // eslint-disable-next-line no-use-before-define
+    // TODO: HumanPlayerCallback,
+  );
+};
+
+const verifyImage = (question) => {
+  if (game.getRunning()) {
+    if (question.image.startsWith('data:image')) {
+      game.addQuestion(question);
+      spreadQuestion(question);
+    } else {
+      // Eslint warn. Function called before declaration.
+      // eslint-disable-next-line no-use-before-define
+      getNewQuestion();
+    }
+  }
+};
+
+const verifyQuestion = (question) => {
+  const interval = setInterval(() => {
+    if (question.image !== undefined) {
+      clearInterval(interval);
+      verifyImage(question);
+    }
+  }, 100);
+};
+
+const getNewQuestion = () => {
+  const question = getQuestion();
+  if (question.err === '' || question.err === undefined) {
+    verifyQuestion(question);
+  } else {
+    getNewQuestion();
+  }
+};
+
+const closeGame = (interval) => {
+  game.changeRunningFlag();
+  clearInterval(interval);
+  // TODO: send_to_modal(game.generateObjectForModal());
+};
+
+const gameRunning = () => {
```

Comment:
Consider using a verb as a first part of the function's name

Reply:
Will be changed in next commit